packages/frontend/src/components/ReactHookForm/InputWrapper.tsx

```diff
@@ -46,7 +48,7 @@ const InputWrapper: FC<InputWrapperProps> = ({
 
     return (
         <FormGroup
-            className="input-wrapper"
+            className={`input-wrapper ${className}`}
```

Comment:
This change allows styled components lib to apply classnames to our form inputs.